### PR TITLE
Draft oneway

### PR DESCRIPTION
**Pull request by wtedeschi**
Hi Kurt,

Here we have two bug fixes and 2 new functionalities:
1. Log is enabled also on the mirror side
2. Oneway latency is included with the logging.

For one way i have tested also the AO and AE security, both worked with the current code.

Thanks,
/Fred

### CODE REVIEW

File: src/rtc_thread.c [hunk 1]
```diff
@@ -104,6 +105,7 @@ static int rtc_gen_and_send_frames(struct thread_context *thread_context, int so
 		frame_config.frame_length = app_config.rtc_frame_length;
 		frame_config.num_frames_per_cycle = app_config.rtc_num_frames_per_cycle;
 		frame_config.sequence_counter = sequence_counter_begin + i;
+		frame_config.tx_timestamp = ts_to_ns(&tx_time);
```
Comment (shifty91) [inline src/rtc_thread.c:108]: This `tx_time` is always zero. It misses the call to `clock_gettime()`

File: src/stat.c [hunk 2]
```diff
@@ -200,50 +228,64 @@ static void stat_frame_received_per_period(enum stat_frame_type frame_type, uint
...
-			fclose(file_trace_marker);
-			exit(EXIT_SUCCESS);
-		}
+	oneway_time = curr_time - tx_timestamp;
+	oneway_time /= 1000;
+
+	stat_update_min_max(oneway_time, &stat->oneway_min, &stat->oneway_max);
+
+	if (stat_frame_type_is_real_time(frame_type) && oneway_time > rtt_expected_rt_limit / 2)
+		stat->oneway_outliers++;
```
Comment (shifty91) [inline src/stat.c:266]: Nice, outliers are detected. However, they're not logged.

**Review by shifty91 — CHANGES_REQUESTED** [review]
Hi Fred,

nice work. Can you address the following overall issues? Then, I'll merge it.

- Adjust the Tx time stamp on `mirror` side (to make one way measurements symmetrical)
- Run `clang-format`
- Fix build errors and warnings
- Stick to RCT

**Comment by wtedeschi** [comment]
Hi Kurt,

Thanks for the quick review!
I have addressed all the comments, I will be testing by Monday and submitting an update.

Best regards,
/Fred

**Comment by shifty91** [comment]
@wtedeschi Applied, thank you.

**Pull request closed without merging**